### PR TITLE
143455/145572 - 0.4 version enccontent publishing json events

### PR DESCRIPTION
**Pull request by harshda15125**
- Added Sap Action Configurations
- Added JSON to XML mapping
- Removed Polygon Section From POCO/DTO objects
- Added Unit Test cases
- Added/Modified Functional Test cases
- Removed Polygon Section From UT's/FT's

### CODE REVIEW

**Review by pravinvarade10801 — APPROVED** [review]
LGTM

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs [hunk 1]
```diff
@@ -130,7 +127,7 @@ public static async Task<bool> CheckXMLAttributes(JsonPayloadHelper jsonPayload,
             return false;
         }
 
-        private static bool? VerifyUpdateAVCSUnitOfSale(string childCell, ZMAT_ACTIONITEMS item, string permitState)
+        private static bool? VerifyUpdateAVCSUnitOfSale(string childCell, ZMAT_ACTIONITEMS item, string permitState, string correctionTag)
```
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs:130]: Please rename method name

**Review by Sadha1501493 — CHANGES_REQUESTED** [review]

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs (hunk 1 above)
Comment (Pradeep15558) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs]: Rename the method name

**Review by Sadha1501493 — APPROVED** [review]
LGTM

**Review by pravinvarade10801 — APPROVED** [review]

**Review by jrippington — APPROVED** [review]
Looks good

**Pull request merged**